darwin/future/core/items/delete_items.py

```diff
@@ -10,7 +11,8 @@
 def delete_list_of_items(
     client: ClientCore,
     team_slug: str,
-    dataset_ids: int | List[int],
+    dataset_ids: int | list[int] | Literal["all"],
+    item_ids: List[UUID],
```

Comment:
changed how delete_items works, item_ids is required so it doesn't make sense to have that as part of the optional filters rather than a required component. Also added in Literal "all" which should delete everything in a dataset_id. I think this still works with item_ids being an empty list but unsure on this corner case as I only just thought of it and haven't tested.

Reply:
I've been able to use the [delete endpoint](https://docs.v7labs.com/reference/items-delete) successfully while not passing in any `item_ids`

In PY-560, I removed item_ids from the core API methods because it wasn't required. Am I missing something?

Reply:
Hmm, I swear in my testing the other day when adding in delete functionality that I got error responses from the endpoint when item_ids weren't included. I think they might have been 200 returns, but the body had an error code in it.

Reply:
If you only provide dataset_ids, you'll get something like:

```
{
  "errors": {
    "code": "MISSING_SELECT_ALL_FILTER",
    "detail": null,
    "message": "Filter object has to contain at least one non-required filter or 'select_all: true'.",
    "status": 422
  }
}
```

But all you need to do is add any other filter, it can be item_ids or anything else

Reply:
ah ok, I'll make that change then. I must have just inferred that an ID had to be present